facebookads/adobjects/adreportrun.py

```diff
@@ -113,14 +116,8 @@ def get_insights(self, fields=None, params=None, async=False, batch=None, pendin
         if async:
           return self.get_insights_async(fields, params, batch, pending)
         param_types = {
-            'default_summary': 'bool',
-            'fields': 'list<fields_enum>',
-            'filtering': 'list<Object>',
-            'sort': 'list<string>',
-            'summary': 'list<summary_enum>',
```

Comment:
I created thie [PR](https://github.com/Work4Labs/work4us/pull/12649) to see if it breaks any unit tests but since everything is mocked I don't think it will change much